Sources/Crossroad/PatternURL.swift

```diff
@@ -44,3 +44,12 @@ internal struct PatternURL {
         return patternString.lowercased().hasPrefix(url.absoluteString.lowercased())
     }
 }
+
+extension PatternURL: Equatable {
+    static func == (lhs: Self, rhs: Self) -> Bool {
```

Comment:
Do not implement `==` directly.

You can implement `hash(into:)` instead.
Please conform to `Hashable`.
https://developer.apple.com/documentation/swift/hashable#

Reply:
Thank you for your review! 
Fixed https://github.com/giginet/Crossroad/pull/32/commits/c91078d96791c7f6c899d00a455fcbbae6b98262